topics/contributing/tutorials/fair-by-design/tutorial.md

```diff
@@ -0,0 +1,1065 @@
+---
+layout: tutorial_hands_on
+title: FAIR-by-Design methodology
+abbreviations:
+  FAIR: Findable, Accessible, Interoperable, Reusable
+  OER: Open Educational Repositories
+  EOSC: European Open Science Cloud
+  CC: Creative Commons
+  MVS: Minimum Viable Skills
+  TeSS: Training eSupport System
+zenodo_link: https://zenodo.org/records/11548062
+questions:
+  - How to develop FAIR learning materials?
+  - How to incorporate the FAIR principles in the instructional design process?
+objectives:
+  - Design FAIR learning materials
+  - Structure FAIR learning materials
+  - Create and publish FAIR-by-Design learning materials
+  - Assess FAIR-ness of learning objects
+time_estimation: 20M
+key_points:
+  - The FAIR-by-Design Methodology ensures that the output of the instructional design process are FAIR learning materials.
+  - Each stage of the FAIR-by-Design Methodology incorporates specific parts of the FAIR principles.
+  - The ouput of the FAIR-by-Design Methodology are FAIR learning materials from both the perspective of instructors and of learners.
+tags:
+  - 'FAIR-by-Design Learning Materials'
+  - "FAIR Learning Objects"
+  - FAIR-by-Design Methodology
+priority: 1
+contributions:
+  authorship:
+    - sonjafiliposka
+    - amisev1971
+    - CarolinLeister
+    - korvoj
+    - domlgreen
+    - isouyioul
+    - achim-kit
+    - agCleo
+    - CRHAD26
+  funding:
+    - skills4eosc
+follow_up_training:
+ - type: external
+   title: Skill4EOSC FAIR-by-design Methodology Training of Trainers
+   link: "https://learning.skills4eosc.eu/course/view.php?id=19"
+lang: "en"
+level: "Introductory"
+extra:
+  description: "Short how to guide that guides you through the stages of the FAIR-by-Design methodology without any specific choice on tools and formats."
+  target_audience: "Instructional designers"
+  URL_resource_type: "Permanent URL"
+  learning_resource_type: "self-paced microlearning unit"
+  access_cost: "No"
+  attribution: "This microlearning unit is based on Filiposka, Sonja, Mishev, Anastas. (2024). FAIR-by-Design Microlearning (1.0.0). https://doi.org/10.5281/zenodo.11548062"
+---
+
+
+Welcome to the FAIR-by-Design Methodology Microlearning GTN adapted tutorial.
+
+> <agenda-title></agenda-title>
+>
+>Below are the main stages of the FAIR-by-Design Methodology {% cite filiposka_2023_8305540 %} that will help guide you on your journey of creating FAIR learning materials.
```

Comment:
The idea was to make a version of the original FBD micro-learning unit that will fit the GTN format. The built-in Agenda box only shows the headings, but the original micro-learning unit needs to have this enriched with key points for each heading. We always aim to keep this tutorial and the original micro-learning unit as similar as possible for easier maintenance.

Reply:
As a compromise, could you either:

1. update the section headings so that the autogenerated agenda resembles more closely how you want it to look, or
2. keep the autogenerated GTN-style agenda, but follow it up with a details or comment box labeled "learning objectives" where you provide details about the learning objectives of each section